var/spack/repos/builtin/packages/root/package.py

```diff
@@ -31,6 +31,8 @@ class Root(CMakePackage):
     # Development version (when more recent than production).
 
     # Production version
+    version("6.26.10", sha256="8e56bec397104017aa54f9eb554de7a1a134474fe0b3bb0f43a70fc4fabd625f")
```

Comment:
Need at least this release for python 3.11+ : https://github.com/root-project/root/issues/11713 .

Reply:
Similar for macOS 13 (though I'm not sure if we can set OS version requirements in Spack) : https://github.com/root-project/root/issues/11714

Reply:
I am wondering, though, whether from a community contribution workflow perspective it doesn't make sense to model these issues with upper limits on dependencies for older version. I.e. we could use `depends_on("python@:3.10")` until someone can verify that 3.11 works with a certain version. At that point they can add `depends_on("python@:3.11")` and modify `depends_on("python@:3.10", when=":6.26.06")`.

Even though I currently don't have the time to figure out the dependencies (and test it, that's what takes most of the time), this still leaves the current package with the same issue: the most recent ROOT version is not working with the most recent python version (not the preferred version).